src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> covertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        for (int i = 1; i < data.size(); i++) {
+            String[] parts = data.get(i).split(",");
+            FruitTransaction.Operation operation =
+                    FruitTransaction.Operation.fromCode(parts[0].trim());
+            String fruit = parts[1].trim();
+            int quantity = Integer.parseInt(parts[2].trim());
```

Comment:
It's good practice to handle NumberFormatException that can occur if 'parts[2]' is not a valid integer. You could wrap the parsing in a try-catch block and throw a more informative RuntimeException if parsing fails.